README.md

```diff
@@ -166,9 +166,120 @@ receivers:
   - service_key: <team-DB-key>
 ```
 
+## Amtool
+
+`amtool` is a cli tool for interacting with the alertmanager api. It is bundled with all releases of alertmanager.
+
+### Install
+
+Alternatively you can install with:
+```
+go get github.com/prometheus/alertmanager/cmd/amtool
```

Comment:
As we ship `amtool` with every release it's most appropriate to describe that it is included in the release tarball IMO, as we have absolutely no guarantee regarding compatibility across versions. So it's safest to just use the amtool that was shipped with the release you're using.